src/main/kotlin/com/mjucow/eatda/domain/banner/service/query/dto/BannerDto.kt

```diff
@@ -0,0 +1,24 @@
+package com.mjucow.eatda.domain.banner.service.query.dto
+
+import com.mjucow.eatda.domain.banner.entity.Banner
+import java.time.Instant
+
+data class BannerDto(
+    val id: Long,
+    val link: String,
+    val imageAddress: String,
+    val expiredAt: Instant?,
+    val createdAt: Instant,
+) {
+    companion object {
+        fun from(domain: Banner): BannerDto {
+            return BannerDto(
+                domain.id,
+                domain.link,
+                domain.imageAddress,
+                domain.expiredAt,
+                domain.createdAt
```

Comment:
정의할 때만 트레일링 콤마 붙고 호출할 때는 안 됐던 것 같기도?
```suggestion
                domain.createdAt,
```

Reply:
트레일링 코마 가능
ktlint 설정 변경 필요